bootstrap_cfn/stacks/ec2.json

```diff
@@ -8,11 +8,26 @@
         "SecurityGroupIngress": []
     }
   },
+  "SaltSG": {
+    "Type": "AWS::EC2::SecurityGroup",
+    "Properties": {
+        "VpcId" : { "Ref" : "VPC" },
+        "GroupDescription": "Salt Comms Security Group",
+        "SecurityGroupIngress": [
+                                    {
+                                        "IpProtocol": "tcp",
+                                        "FromPort": 4505,
+                                        "ToPort": 4506,
```

Comment:
Just to check: is this the right way to define the port range, or does it map 4505 to 4506?